CONTRIBUTING.md

```diff
@@ -68,6 +68,16 @@ Updating an existing devfile stack is relatively straightforward:
     - Minimally, testing with odo v2 (`odo create`, `odo push`, etc) and odo v3 (`odo init`, `odo dev`, etc) is recommended, however if your Devfile is used with other tools, it's recommended to test there as well.
```

Comment:
I would propose mentioning testing with **Che** as a minimal requirement for contributing:

> - Minimally, testing with **odo v2** (`odo create`, `odo push`, etc) and **che** (see "[How to Test Changes > Che](https://github.com/devfile/registry/blob/main/CONTRIBUTING.md#che)" for details); **odo v3** (`odo init`, `odo dev`, etc) is recommended, however if your Devfile is used with other tools, it's recommended to test there as well.

Reply:
@dmytro-ndp done